Lib/idlelib/idle_test/run_coverage.py

```diff
@@ -0,0 +1,135 @@
+import argparse
+import os
+import pathlib
+import subprocess
+import sys
+import textwrap
+
+
+def main():
+    idlelib_path = find_idlelib_path()
+    sources_root_path = idlelib_path.parent.parent
+    venv_path, venv_python_path = ensure_venv(sources_root_path)
```

Comment:
This should be moved after the args have been parsed so we don't bother installing if just --help is called

Reply:
Good call!